functions/tests/test_stripe_webhooks.py

```
@@ -0,0 +1,560 @@
+        mock_check_processed.return_value = False
+        mock_fulfill.return_value = "order-123"
+        mock_send_email.return_value = True
+        mock_https_fn.Response.return_value = Mock(status=200)
+
+        # Mock line items and customer
+        mock_line_items = Mock()
+        mock_customer_details = Mock()
+        mock_customer_details.email = "test@example.com"
+        mock_checkout_session = Mock()
+
+        result = fulfilment_workflow_on_ticket_purchase(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)